test/e2e/config/docker.yaml

```diff
@@ -159,31 +159,31 @@ providers:
     files:
     - sourcePath: "../data/shared/v1.0/metadata.yaml"
     - sourcePath: "../data/infrastructure-docker/v1.0/cluster-template.yaml"
-  - name: v1.3.6 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.3.6/infrastructure-components-development.yaml"
+  - name: v1.4.5 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.4.5/infrastructure-components-development.yaml"
     type: "url"
     contract: v1beta1
     replacements:
     - old: --metrics-addr=127.0.0.1:8080
       new: --metrics-addr=:8080
     files:
-    - sourcePath: "../data/shared/v1.3/metadata.yaml"
-    - sourcePath: "../data/infrastructure-docker/v1.3/cluster-template.yaml"
-    - sourcePath: "../data/infrastructure-docker/v1.3/cluster-template-topology.yaml"
-    - sourcePath: "../data/infrastructure-docker/v1.3/clusterclass-quick-start.yaml"
```

Comment:
Question: I updated all v1.3.6 release mentions to v1.4.5, and deleted `"../data/shared/v1.3/metadata.yaml"`. But now I see other files under `../data/infrastructure-docker/v1.3` are also not needed anymore since there is no file referencing them. Should we remove  `../data/infrastructure-docker/v1.3` folder in that case?

Reply:
/cc @sbueringer @fabriziopandini

Reply:
> But now I see other files under ../data/infrastructure-docker/v1.3 are also not needed anymore since there is no file referencing them. Should we remove ../data/infrastructure-docker/v1.3 folder in that case?

checked again, it is used in Makefile target to generate e2e templates https://github.com/kubernetes-sigs/cluster-api/blob/main/Makefile#L516-L519.